docs/field_types.md

```diff
@@ -25,6 +25,7 @@ Currently the following field types are supported:
 | Decimal32Field     | Decimal32  | Decimal             | Ditto
 | Decimal64Field     | Decimal64  | Decimal             | Ditto
 | Decimal128Field    | Decimal128 | Decimal             | Ditto
+| UUIDField          | UUID       | Decimal             | 
 | Enum8Field         | Enum8      | Enum                | See below
```

Comment:
I'm not sure whether Pythonic type should be `Decimal` , why not `uuid.UUID` ?

Reply:
@emakarov of course `uuid.UUID`, my bad, I cloned the line above and didn't change Pythonic type